### PR TITLE
get_manifest is now optionnal

### PR DESCRIPTION
**Pull request by martigan**
paramètre pour rendre la récupération du manifest optionnel

authentification sur la requete https pour récupérer le manifeste

### CODE REVIEW

File: src/grocker/builders/op.py [hunk 1]
```diff
@@ -98,7 +100,11 @@ def get_manifest_digest(name):
         return None  # Docker HUB API is not documented
 
     registry, repository = registry_repository.split('/', 1)
-    response = requests.head('https://{}/v2/{}/manifests/{}'.format(registry, repository, tag))
+    auth_info = None
+    with open('.docker/config.json', 'r') as config_file:
```
Comment (xavfernandez) [inline src/grocker/builders/op.py:104]: I'm not sure this solution is general enough: the file might not exist or exist in an other place depending on the OS, installation, etc. Let's forget this part and simply keep the manifest option.

File: src/grocker/__main__.py [hunk 2]
```diff
@@ -80,8 +80,11 @@ def purge(all_versions, including_final_images):
     '--push/--no-push', default=True,
     help='push the image',
 )
+@click.option(
+    '--manifest/--no-manifest', default=True, help='Hash provided from manifest',
+)
 @click.argument('release')
-def build(release, build_dependencies, build_image, push, **kwargs):
+def build(release, build_dependencies, build_image, push, manifest, **kwargs):
```
Comment (xavfernandez) [inline src/grocker/__main__.py:87]: I'd like to also be able to configure it through our shared configuration file.

**Review by xavfernandez — CHANGES_REQUESTED** [review]

File: src/grocker/__main__.py (hunk 2 above)
Comment (martigan) [reply src/grocker/__main__.py]: done, set to false by default

**Review by xavfernandez — APPROVED** [review]
A changelog is also needed :)

**Pull request merged**